### DISCUSSION TITLE
Create session from verify email URL token

### DISCUSSION THREAD

**Discussion by tawjaw** [supabase/supabase#733] [Questions] [answered]
Hello
After signing up I get information in the link parameters from the verification email.
How can I then log in the user using the client library?

Similarly for the magic link logging in.

Maybe I missed it, but I couldn't find in the documentation anything related to the email URL parameters and how to use them in the client library. 

Thanks

**Comment by kiwicopple** [comment, marked as answer]
Hey @tawjaw , it might be easiest to check one of the Todo App examples here: 
https://github.com/supabase/supabase/tree/master/examples

As long as you have the Email links redirect the user to a page on your site, the supabase-js library with "detect" the session in the URL.

**Reply by tawjaw** [reply]
Thanks @kiwicopple !
I've looked at the examples which raised my question.. it only handles what is the type of the URL parameters and redirect based on that.. but the app doesn't do anything with the session token. 

I now realize this option in the library 

`detectSessionInUrl: true,`

Thanks again for the quick reply!
